templates/CRM/common/l10n.js.tpl

```diff
@@ -128,5 +128,11 @@
     params: {},
     functions: []
   };
+
+  // Load polyfill
+  if (!('Promise' in window)) {
+    CRM.loadScript(CRM.config.resourceBase + 'bower_components/es6-promise/es6-promise.auto.min.js');
```

Comment:
Generally, :+1: on better access to standardized promises.

I don't actually know the right place to put this, and this prompt isn't mean to be a hard-block, but it feels a little odd to be polyfilling the `Promise` API for use-by Paypal SDK from a file named `l10n.js.tpl`. It may actually be the right thing... but it feels weird.